### DISCUSSION TITLE
Diployed Whisper with FastApi but its throwing an Error cannot find ffmpeg

### DISCUSSION THREAD

**Discussion by JenuelDev** [openai/whisper#1958] [Q&A]
So, the apt ffmpeg is already installed but when deployed in development, it cannot find the ffmpeg. But if working with local, it can find the ffmpeg.

**Comment by phineas-pta** [comment]
more like problem with your setup than problem with whisper

**Reply by JenuelDev** [reply]
I found the answer now, sory it seems that my problem was in my nginx conf problem.
